src/index.html

```diff
@@ -11,7 +11,226 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="form-block">
+
+        <legend>Personal information</legend>
+        <div class="block-input">
+          <label for="surname">
+            Surname:
+          </label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="block-input">
+          <label for="name">
+            Name:
+          </label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </div>
+
+
+        <div class="block-input">
+          <label for="year">
+            How old are You?
+          </label>
+          <input
+            type="number"
+            name="year"
+            id="year"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="block-input">
+          <label for="birthday">
+            Full date of birth:
+          </label>
+          <input
+            type="date"
+            name="birthday"
+            id="birthday"
+          >
+        </div>
+
+        <div class="block-input">
+          <label for="agreement">
+            I accept the term of the agreement
+          </label>
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-block">
+
+        <legend>
+          Registration
+        </legend>
+        <div class="block-input">
+          <label for="mail">
+            E-mail:
+          </label>
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="mail"
+            id="mail"
+            required
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="block-input">
+          <label for="pass">
+            Password:
+          </label>
+          <input
+            type="password"
+            name="pass"
+            id="pass"
+            required
+            minlength="8"
+            maxlength="32"
+            pattern="\S+"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-block">
+
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div class="block-input">
+            Do you love cats?
```

Comment:
it should be in some tag (span, for example)
```suggestion
            Do you love cats?
```